test/implementations/nlp/IncrementalVocabulary.test.ts

```diff
@@ -0,0 +1,25 @@
+import { IncrementalVocabulary } from  "crunchDB/implementations";
+import {  Word, WordRoot } from "crunchDB/objects";
+describe("IncrementalVocabulary", () => {
+  const words: (Word | WordRoot)[] = [Word("apple"), Word("banana"), Word("orange")];
```

Comment:
Don't use a shared object between unit tests. Instead, create a class IncrementalVocabularyMocks that has the values you need. Then, in each individual test, do a `const mocks = new IncrementalVocabularyMocks();` The reason why is that is prevents coupling between the tests and allows them to be run in any order, which is important for unit tests.

Reply:
I have used beforeEach() hooks which apply to every test in a file and since the initial setup is done inside that hook so changing the order of test won't impact any other test.